### PR TITLE
HIR-143. Add an endpoint to get accommodation details

### PR DESCRIPTION
**Pull request by oleg1std**
Add an endpoint to get accommodation details

### CODE REVIEW

File: HappyTravel.Hiroshima.DirectManager/Services/AccommodationManagementService.cs [hunk 1]
```diff
@@ -90,6 +90,20 @@ async Task<List<Accommodation>> GetAccommodationsWithRooms(int serviceSupplierId
         }
 
 
+        public async Task<Result<Accommodation>> GetInternal(int accommodationId)
+        {
+            var accommodation = await _dbContext.Accommodations
+                .Include(a => a.Location)
+                .ThenInclude(location => location.Country)
```
Comment (DFSko) [inline HappyTravel.Hiroshima.DirectManager/Services/AccommodationManagementService.cs:97]: Maybe `.Include(a => a.Location.Country)`

File: HappyTravel.Hiroshima.WebApi/Services/AvailabilitySearch/AccommodationResponseService.cs [hunk 2]
```diff
@@ -38,6 +60,84 @@ private string GetName(Accommodation accommodation, string languageCode)
         }
 
 
+        private List<string> GetAmenities(Accommodation accommodation, string languageCode)
+        {
+            accommodation.AccommodationAmenities.TryGetValueOrDefault(languageCode, out var amenities);
```
Comment (DFSko) [inline HappyTravel.Hiroshima.WebApi/Services/AvailabilitySearch/AccommodationResponseService.cs:65]: I think better to add GetValueOrDefault method

File: HappyTravel.Hiroshima.DirectManager/Services/AccommodationManagementService.cs (hunk 1 above)
Comment (oleg1std) [reply HappyTravel.Hiroshima.DirectManager/Services/AccommodationManagementService.cs:97]: No

File: HappyTravel.Hiroshima.WebApi/Services/AvailabilitySearch/AccommodationResponseService.cs (hunk 2 above)
Comment (oleg1std) [reply HappyTravel.Hiroshima.WebApi/Services/AvailabilitySearch/AccommodationResponseService.cs:65]: There is a possible situation when the default language is not defined.
TryGetValueOrDefault  is a more suitable name, in my opinion,

**Review by hasmikharutyunyan — APPROVED** [review]

**Review by AlexeyShok — APPROVED** [review]

**Pull request merged**